packages/reactive-element/src/decorators/query-assigned-elements.ts

```diff
@@ -67,7 +81,8 @@ export function queryAssignedElements(options?: QueryAssignedElementsOptions) {
         const slotSelector = `slot${slot ? `[name=${slot}]` : ':not([name])'}`;
         const slotEl =
           this.renderRoot?.querySelector<HTMLSlotElement>(slotSelector);
-        const elements = slotEl?.assignedElements(options) ?? [];
+        const elements =
+          slotEl != null ? slotAssignedElements.call(slotEl, options) : [];
```

Comment:
I suggest making this take an element so you don't have to use call. It's simpler and should be just slightly faster.

Reply:
Done! Thank you!